packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -249,10 +234,48 @@ export const Product = ({
       <h3 className="billing-title">
         <span>Billing Information</span>
       </h3>
+
+      <TransitionGroup component={null}>
+        {createTokenError.error && (
+          <CSSTransition classNames="animate-fade" timeout={150}>
```

Comment:
This seems gross to me because I'd like to not have to specify `animate-fade` with `timeout={150}` every time, but if we want to control the conditional in _this_ file and use an animation on `ErrorMessage`, we need `CSSTransition` in the same place as the conditional and `TransitionGroup` around the whole thing. (At least, this was the only way I could get it working.)

`TransitionGroup` sticks this all in a `div` by default, `component={null}` just says "don't add an extra container component".

Reply:
Could we move the repeated `TransitionGroup` and `CSSTransition` boilerplate into the new `ErrorMessage` component - but maybe add an `visible` prop for the conditional (i.e. `visible={createTokenError.error}` and use `{visible && ( <CSSTransition...` in `ErrorMessage`? Seems like that would render as nothing if the only thing returned is an empty `TransitionGroup`